trade-finance-manager-ui/templates/case/underwriting/amendments/amendment-lead-underwriter/amendment-lead-underwriter.njk

```diff
@@ -0,0 +1,51 @@
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% macro render(params) %}
+  {% set leadAmendmentUnderwriter = params.amendmentLeadUnderwriter %}
+
+  {% block underwriting_content %}
+
+    {% if leadAmendmentUnderwriter.currentLeadUnderWriter %}
+
+      <div class="govuk-grid-row">
+
+        <div class="govuk-grid-column-three-quarters">
+          <p class="govuk-!-margin-top-0" data-cy="amendment-lead-underwriter-name">{{ leadAmendmentUnderwriter.amendment.amendments.leadUnderwriter.firstName }} {{ leadAmendmentUnderwriter.amendment.amendments.leadUnderwriter.lastName }}</p>
```

Comment:
can we simplify the way we pass these variables? it becomes very long in the current format
something like: `leadUnderwriter.firstName`

Reply:
done